conf/flight_plans/demo_gvf.xml

```diff
@@ -1,21 +1,35 @@
-<!DOCTYPE flight_plan SYSTEM "flight_plan.dtd">
```

Comment:
that doesn't look correct, the `flight_plan.dtd` is in the same directory as the flightplan, so it was correct before